src/layouts/homepage.html.jade

```diff
@@ -34,19 +34,20 @@ html(lang="en")
                         span.title Cozy&nbsp;
                         | a Personal Cloud
                         br
-                        | You can Host, Hack
-                        | and Delete
-                    h4
-                        | Your contacts, calendars and files are stored on your
-                        | hardware and synced with your devices
+                        | you can host, customize
+                        | and fully control
```

Comment:
The issue is both with "hack" and "delete". Both are negative and therefore have a negative impact in perceiving the brand. "Hack" means cracking for many people (I think it should not, but that's a fact) while the perspective of having my personal server "hacked" is quite frightening. We should emphasize the fact that on Cozy the user's data is safe. 

Also, "delete" does not mean a lot in this context. I understand that you refer to the fact that people can't delete their data on Facebook, but it's not obvious and the last thing I want in most cases is my data to be deleted.. "Control" is much better in this regard. 

If you have better suggestion for replacing "customize", I'm all ears. "Personalize"?